stackslib/src/chainstate/nakamoto/tenure.rs

```diff
@@ -197,6 +197,56 @@ pub static NAKAMOTO_TENURES_SCHEMA_2: &'static str = r#"
     CREATE INDEX nakamoto_tenures_by_parent ON nakamoto_tenures(tenure_id_consensus_hash,prev_tenure_id_consensus_hash);
 "#;
 
+pub static NAKAMOTO_TENURES_SCHEMA_3: &'static str = r#"
+    -- Drop the nakamoto_tenures table if it exists
+    DROP TABLE IF EXISTS nakamoto_tenures;
+
+    -- This table records each tenure-change, be it a BlockFound or Extended tenure.
+    CREATE TABLE nakamoto_tenures (
```

Comment:
```suggestion
    CREATE TABLE nakamoto_tenures STRICT (
```

Reply:
The STRICT will become available after this PR gets merged https://github.com/stacks-network/stacks-core/pull/4948. It is in review. The STRICT keyword could be added for all new created tables.

ref https://github.com/stacks-network/stacks-core/pull/4948#issuecomment-2210882174

Reply:
Yup, added